sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -435,6 +482,12 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-tgl-rt715-rt711-rt1308-mono.tplg",
 	},
+	{
+		.link_mask = 0xF, /* 4 active links required */
```

Comment:
can you remind me why we need the 4 links? This is UpExtreme so presumably you are using link0 only, no?

Reply:
> can you remind me why we need the 4 links? This is UpExtreme so presumably you are using link0 only, no?

I use UpExtreme  indeed, and enable 4 links via selecting AAM to SoundWire.